index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <style>
+      @import url("http://fonts.cdnfonts.com/css/cocogoose");
+    </style>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/71d90399a2.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Capstone - home</title>
+  </head>
+  <body>
+    <header>
+      <nav class="topnavbar">
+        <div></div>
+        <ul class="topnavbaricons">
+          <li><i class="fab fa-facebook-f"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li>english</li>
+          <li>my page</li>
+          <li>logout</li>
+        </ul>
+      </nav>
+      <nav class="secnavbar">
+        <ul>
+          
+            <li><a href="index.html"><img class="logo" src="assets/images/logo.png" alt="" /><div class="logoli"></div></a>
+          
+          <li class="secmenuright">
+            <ul class="secmenu">
+              <li><div class="closebutton small"><i class="fas fa-times"></i></div></li>
+              <li><a class="aoutlink" href="about.html">About</a></li>
+              <li>Program</li>
+              <li>Join</li>
+              <li>Sponsor</li>
+              <li>News</li>
+            </ul>
+            <button class="ccbutton">cc campain</button>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] I do see something on the left of the mobile menu, please correct it. and also the display center for the links are a bit deviated.

![Inkedmob](https://user-images.githubusercontent.com/61562626/137174062-8afd2ef8-01f1-460a-8c6c-9187a51e9cd5.jpg)